### ISSUE TITLE
Advertise support for languages other than just Markdown

### ISSUE THREAD

**Issue by jansol** [koozz/zed-vale#4]
The extension.toml should ideally register the language server for all languages listed in https://vale.sh/docs/topics/scoping/#formats

**Comment by koozz** [comment]
You're right, but it's quite a list. I'll see what I can do.

**Comment by jansol** [comment]
Yeah, it would be nice if it was possible to just specify `*` or something to unconditionally load the extension for all files.

**Comment by p0fi** [comment]
Hi, this issue is a bit old. I would also like to have Vale enabled for more languages, especially AsciiDoc. @koozz Could you add some more languages, if not the entire list?

**Comment by koozz** [comment]
I haven't tested `*`, but `AsciiDoc` and `Plain text` are added. Now the zed-extensions repository needs to bump the git-submodule to the new commit.

Though a quick test seems like vale needs asciidoctor to check on `adoc`, @p0fi do you have any experience with that?

**Comment by ofek** [comment]
@koozz Are you able to enable the rest as well? There don't appear to be that many and whether they work out-of-the-box is the users' concern rather than the extension's (aside from a note in the docs).

<img width="1165" height="730" alt="Image" src="https://github.com/user-attachments/assets/03aa2abe-cc66-44a0-8c2b-a2d32411141c" />

**Comment by koozz** [comment]
Hi @ofek,

If you have the correct Zed identifiers for these types, I can add them, or even happily accept a pull request.

I haven't used Zed for a while now. So I'm also open for project transfer if anyone feels up for it.

**Comment by ofek** [comment]
Sure, here's what I could find:

- **`HTML`** - https://github.com/zed-industries/zed/blob/main/extensions/html/languages/html/config.toml
- **`MDX`** - https://github.com/srazzak/zed-mdx/blob/main/languages/mdx/config.toml
- **`Org`** - https://github.com/hron/zed-org/blob/main/languages/org/config.toml
- **`reST`** (reStructuredText) - https://github.com/elmarco/zed-rst/blob/main/languages/rst/config.toml
- **`XML`** - https://github.com/sweetppro/zed-xml/blob/main/languages/xml/config.toml
